.github/workflows/codecov.yml

```diff
@@ -0,0 +1,63 @@
+name: Codecov
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+    types: [opened, reopened, synchronize]
```

Comment:
Do we really want this to run with every single commit added to a PR? It's a 1h test suite. It'll take up a lot of resources on Klas' node.

Reply:
It takes 13min on Klas machine 😃

Reply:
Whooot?!?